delocate/delocating.py

```diff
@@ -526,42 +558,44 @@ def delocate_wheel(
         all_copied = {}  # type: Dict[Text, Dict[Text, Text]]
         wheel_dir = realpath(pjoin(tmpdir, 'wheel'))
         zip2dir(in_wheel, wheel_dir)
-        for package_path in find_package_dirs(wheel_dir):
-            lib_path = pjoin(package_path, lib_sdir)
-            lib_path_exists = exists(lib_path)
-            copied_libs = delocate_path(
-                package_path,
-                lib_path,
-                lib_filt_func,
-                copy_filt_func,
-                executable_path=executable_path,
-                ignore_missing=ignore_missing,
-            )
-            if copied_libs and lib_path_exists:
+        # Assume the package name from the wheel filename.
```

Comment:
Function getting a bit long now.  Can you think of a way of splitting this up?  Don't worry if not - it's hardly your changes that are causing the length.

Reply:
Everything was in one big for loop and taking out the loop caused a dedent on a lot of code.  I thought of splitting it up but in the end I was focusing more on fixing this rather major issue rather than cleaning things up.

Reply:
Yes, no problem.  For a later date, maybe.

Reply:
If I don't clean this up now then I'm not sure it will ever happen.

Reply:
Totally your call.